fedoracommunity/widgets/package/bugs.py

```diff
@@ -20,15 +20,17 @@ class BugStatsWidget(twc.Widget):
     status_closed_string = "bug_status=CLOSED"
 
     base_query_string = twc.Variable(default='')
-
+    
     def prepare(self):
         super(BugStatsWidget, self).prepare()
-        self.base_query_string = urllib.urlencode({
-            "query_format": "advanced",
-            "product": self.product,
-            "component": self.package,
-        })
-
+        # To get more than one key/value pair with the same key in urllib.urlencode, we need to create a tuple first
+        query_product = (
```

Comment:
```suggestion
        self.query_product = (
```

Reply:
Because I had to move `query_product`, I got an error when I first tried this. `query_product` is now defined inside the function definition, not outside. I moved it so that I could get `self.product` set for the `component` key. I did not see any need that is satisfied by having it outside of the function definition, but I could be wrong.

Reply:
I applied the change for testing. It's entirely possible that the `500` error I encountered the first time could have been caused by something else.